WordPressKitTests/RemoteTestCase.swift

```diff
@@ -129,8 +130,10 @@ extension RemoteTestCase {
     ///         https://github.com/AliSoftware/OHHTTPStubs/wiki/Usage-Examples#stack-multiple-stubs-and-remove-installed-stubs
     ///
     func stubAllNetworkRequestsWithNotConnectedError() {
-        stub(condition: { _ in
-            return true
+        stub(condition: { request in
+            // Stub all requests other than those to the Buildkite Test Analytics API.
+            // We need those to go through for Test Analytics reporting.
+            request.url != TestCollector.apiURL
```

Comment:
I see there are a couple places doing similar check, I'm wondering is there anyway to bypass this buildkite API URL globally?

Reply:
I looked into OHHTStubs but couldn't find any global setup hooks. Luckily we have @AliSoftware himself that we can ask this to 😄 .

Reply:
I don't know whether we need a fallback stub like in the setup you describe.

What I wish we could do is saying "always allow requests to `isHost(TestCollector.apiHost)` to go through". But I can see how this is not part of the library's philosophy and also not feasible in vanilla XCTest, where there is no "before the test suite runs" hook.

And for good reason!

It might be extra work to setup exceptions on a per-test-case basis, but the alternative would be to open the global state flood gates in the tests, and that would be worse, leading to all sorts of bad test design and non-deterministic failures.